app/models/product_import/entry_validator.rb

```diff
@@ -4,6 +4,7 @@
 
 module ProductImport
   class EntryValidator
+    # rubocop:disable Metrics/ParameterLists
```

Comment:
I think this should be dealt with. Either in a tech-debt task or in this pull request, I don't mind. But I agree with Pau that this is really bad style. It's like having global variables in PHP. ;-) Not blaming you. It's just a growth problem to deal with.